### DISCUSSION TITLE
How to set the location for amazon as US?

### DISCUSSION THREAD

**Discussion by lozbinks** [jef/streetmerchant#2429] [Help]
When scanning Amazon US it will have my location set as UK which causes a lot of available stock to not show up and therefore not be detected.

I've set my location in the dotenv as "USA" but amazon still shows my location as UK..

I've used a US VPN but amazon still shows my location as UK..

I've tried logging in on chromium and setting location to US on amazon but after about 5 mins it logs me out and sets my location to UK..

**Comment by Mr-Tech-13** [comment]
Are you in the uk?

**Reply by lozbinks** [reply]
I am yes

**Reply by lozbinks** [reply]
Using a US VPN does seem to work now however it is really really slow at scanning just amazon compared to how fast it scans amazon-uk without a VPN

**Comment by Mr-Tech-13** [comment]
Using a VPN will be slower because it has to travel through some servers before you can access the internet

**Reply by lozbinks** [reply]
I appreciate that, however I'm sure I've tried it without the VPN when it allows me to set my location to US by logging in for about 5 minutes and if I remember correctly I'm sure it still scans far far slower than it scans amazon-uk.

Are you aware of anyway to set the location to remain as US without using a VPN?

**Reply by Mr-Tech-13** [reply]
Not that I can think of

**Reply by lozbinks** [reply]
Ok, no worries
